autoload/lsp/completion.vim

```diff
@@ -296,6 +296,9 @@ export def CompletionReply(lspserver: dict<any>, cItems: any)
     endif
 
     d.user_data = item
+
+    # Some servers like volar-server sends duplicates
+    completeItems = filter(completeItems, (_, i) => i.abbr != d.abbr && i.kind != d.kind)
```

Comment:
looks would be busy,
and specially dup was allowed and needed, e.g some snip item had same name and same kind,
or even different kind maybe same name.